tasks/skylab/PairedTagUtils.wdl

```diff
@@ -7,7 +7,7 @@ task PairedTagDemultiplex {
         String input_id
         Boolean preindex
         File whitelist
-        String docker = "us.gcr.io/broad-gotc-prod/upstools:1.2.0-2023.03.03-1704723060"
+        String docker_path
```

Comment:
This didn't need to be made backwards compatible since it's only used in PairedTag.wdl currently.